content/install_maintain/installation/installing-cluster.md

```diff
@@ -44,69 +42,14 @@ In case you use an Externally hosted PostgreSQL or RabbitMQ, i.e. "bring your ow
 read the relevant information for this case. 
 {{% /note %}}  
 
-{{% note title="VMs setup" %}}  
-Before you proceed, make sure you have all VMs spinning, that they are all allocated with a public-ip, and that they are configured according
+**Note:** Before you proceed, make sure you have al l VMs spinning, that they are all allocated with a public-ip, and that they are configured according
```

Comment:
Minor typo.
**Note:** Before you proceed, make sure that all the required VMs are spinning, that they are all allocated with a public-IP, and that they are configured according